homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -67,33 +113,17 @@ async def async_setup_entry(
 
     sensor_list = []
     for i, sensor in enumerate(sensors):
-        if not sensor.permissions.get("read"):
-            LOGGER.warning(
-                "No permission to read sensor %s, are you sure you're signed into the right account?",
-                sensor.name,
-            )
-            continue
         for field in sensor.sensor_field_names:
+            description = SENSOR_DESCRIPTIONS.get(field, None)
+            if description is None:
+                message = f"Unsupported sensor field: {field}\nPlease create an issue on GitHub. https://github.com/home-assistant/core/issues/new?assignees=&labels=&template=bug_report.yml&integration_name=LaCrosse%20View&integration_link=https://www.home-assistant.io/integrations/lacrosse_view/&additional_information=Field:%20{field}%0ASensor%20Model:%20{sensor.model}&title=LaCrosse%20View%20Unsupported%20sensor%20field:%20{field}"
+                LOGGER.warning(message)
+                return
+            description.key = str(i)
```

Comment:
This won't work. The sensor descriptions are constants. Store the sensor index as a separate attribute in the entity instead, if needed.